src/cmd/yacc.c

```diff
@@ -1936,6 +1937,11 @@ cstash(char *s)
 	return temp;
 }
 
+int
+isvalidchar(int i) {
+	return (i & ~0x7f) == 0;
```

Comment:
Hmm, I don't know why I made this 0x7f instead of 0xff, whic hseems like it would make more sense.